addons/mail/static/src/models/composer/composer.js

```diff
@@ -509,7 +509,7 @@ function factory(dependencies) {
          * @returns {mail.partner[]}
          */
         _computeMentionedChannels() {
-            const inputMentions = this.textInputContent.match(
+            const inputMentions = this.textInputContent.replace('\n', "\n ").match(
```

Comment:
missing test for channel name wrongly detected when followed directly by a new line